openfermioncirq/gates/fermionic_simulation.py

```diff
@@ -86,11 +88,99 @@ def state_swap_eigen_component(x: str, y: str, sign: int = 1, angle: float=0):
     return component
 
 
+@cirq.value_equality(approximate=True)
+class FermionicSimulationGate(cirq.EigenGate):
```

Comment:
This name is too vague and also can potentially be confused with `cirq.FSimGate`. How about `ParityPreservingFermionicGate`?

Reply:
What about `ParityPreservingFermionicSimulationGate`, so that the name reflects that it's the exponential of a Hamiltonian?

Or `JWStringExpGate`?

Reply:
I don't think the word "Simulation" adds any meaning, and with it the name is too long. The latter name is too obscure, and in any case we have been using the convention that it is unnecessary to specify "JW" explicity.

Reply:
Okay, let's go with `ParityPreservingFermionicGate`. Should we change the names of the children as well (e.g., `QuadraticFermionicSimulationGate` to `QuadraticFermionicGate` or `QuadraticParityPreservingFermionicGate`)? If so, should we deprecate the old names? (They're already in master.)

Reply:
I'm not opposed to renaming the children, but that should be done in a separate PR. And yes, if we do that, we should deprecate the old names.

Reply:
Done.